src/plist.h

```diff
@@ -291,6 +291,46 @@ static inline Obj PopPlist(Obj list)
     return val;
 }
 
+
+/****************************************************************************
+**
+*F  NewEmptyPlist() . . . . . . . . . . . . . . . .  Create a new empty Plist
+*F  NewImmutableEmptyPlist() . . . . . . . Create a new immutable empty Plist
+*/
+static inline Obj NewEmptyPlist(void)
+{
+    return NEW_PLIST(T_PLIST_EMPTY, 0);
+}
+
+static inline Obj NewImmutableEmptyPlist(void)
+{
+    return NEW_PLIST_IMM(T_PLIST_EMPTY, 0);
```

Comment:
How about adding a singleton immutable empty plist, and return that from here, instead of creating lots of new ones? Or rather, since the name `NewImmutableEmptyPlist` then would be misleading, just expose that global here, as `extern Obj ImmutableEmptyPlist;` or so?